docs/source/troubleshooting/general.rst

```diff
@@ -47,8 +47,26 @@ Also, if you have many controllers already directly connected to the host, it mi
 Sunshine provided controller (connected to the guest) is the "first" one. In Linux this can be accomplished on USB
 devices by finding the device in `/sys/bus/usb/devices/` and writing `0` to the `authorized` file.
 
-Packet loss
------------
+Packet loss (Buffer overrun)
```

Comment:
Just above this, can you remove the `Nvidia issues` section?

Reply:
Sure, I thought the recent PR only enabled NvFBC. So it also allows for multiple concurrent encoder sessions?
That's great then.

Reply:
Yea that's right... and Nvidia has raised the NVENC limit on all their cards to 5 if I remember correctly. It used to be 2.